admin/views/tabs/dashboard/features.php

```diff
@@ -59,8 +59,12 @@
 		);
 	}
 
-	$calibration_beta = new WPSEO_Recalibration_Beta();
-	$calibration_beta->show_feature_toggle();
+	// If the recalibration beta has been disabled you will no longer be able to enable it.
+	// See https://github.com/Yoast/wordpress-seo/issues/12183.
+	if ( WPSEO_Recalibration_Beta::is_enabled() ) {
```

Comment:
I would move this logic to the `show_feature_toggle`. This will follow the Tell, don't ask principle.